src/assets/v1.4.1/compatibility_fallback_handler.json

```diff
@@ -340,4 +341,4 @@
       "type": "function"
     }
   ]
-}
+}
```

Comment:
@0xad1onchain could you configure your editor to not remove these final newlines and commit again?

Reply:
Fixed it in https://github.com/safe-global/safe-deployments/pull/344,  https://github.com/safe-global/safe-deployments/pull/345, https://github.com/safe-global/safe-deployments/pull/346 and https://github.com/safe-global/safe-deployments/pull/347